tests/jerry/regression-test-issue-122.js

```diff
@@ -15,10 +15,11 @@
 
 try
 {
-  print(this, []);
+  print({toString: function() { throw new TypeError("foo"); }}, []);
```

Comment:
It is related, a with this PR the global object can be printed out and won't cause an Error. Due to the fact that the prototype is now set, and thus the toString method will be found when converting the object to string.